### PR TITLE
rgw : add error message when rm key

### PR DESCRIPTION
**Pull request by BryceCao**
Fixes: https://tracker.ceph.com/issues/53725
Signed-off-by: caolei <halei15848934852@163.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [x] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/rgw/rgw_user.cc [hunk 1]
```diff
@@ -855,7 +855,9 @@ int RGWAccessKeyPool::execute_remove(const DoutPrefixProvider *dpp,
   map<std::string, RGWAccessKey> *keys_map;
 
   if (!op_state.has_existing_key()) {
-    set_err_msg(err_msg, "unable to find access key");
+    std::string key_type_str = key_type ? "s3" : "swift";
```
Comment (dang) [inline src/rgw/rgw_user.cc:858]: This is an enum, not a boolean, so the boolean compare isn't correct.  Probably a better solution is a little helper function to convert an ObjectKeyType into a string.

Comment (BryceCao) [reply src/rgw/rgw_user.cc]: @dang   Thanks. I have added  a conversion function, by the way, this bug can be reproduced by executing "radosgw-admin key rm --uid ... --subuser ... --access-key ..."

**Review by dang — APPROVED** [review]

**Comment by dang** [comment]
jenkins test api

**Review by cbodley — APPROVED** [review]

**Comment by cbodley** [comment]
passed qa https://pulpito.ceph.com/cbodley-2022-01-28_14:34:56-rgw-wip-cbodley-testing-distro-default-smithi/

**Pull request merged**